### ISSUE TITLE
Question: Could this be used to generate markdown from any table?

### ISSUE THREAD

**Issue by richard-edwards** [ozum/pg-generator#109]
I am looking for a way to query a postgres table and generate a set of markdown files based on one or more templates. I understand that this currently uses the database meta data of tables, fields etc. to generate code files etc. but I was wondering if it had the ability to take a table or view, say 'select * from my_document_view" and run against a generator to build a directory of markdown files that would be consumed by a site builder.

**Comment by ozum** [comment]
No, this is a database reverse engineering tool, so it only supports meta-data (table names, column names, view names, column data types, etc.). It can't use data from table rows.

**Comment by richard-edwards** [comment]
Ok thank you for the quick reply. Any suggestions for something like that? I could probably knock something together quickly but I imagine there has to be something out there already.

**Comment by ozum** [comment]
Unfortunately, I don't know any tool for this purpose.
